### PR TITLE
Bob approaches

### PR DESCRIPTION
**Pull request by bobahop**
# pull request

Approaches for the Bob exercise

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

**Comment by bobahop** [comment]
@ErikSchierboom : Java Bob approaches ready for review. I put code into an online Java code formatter and checked against [coding conventions](https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#449)

File: exercises/practice/bob/.approaches/if-statements/snippet.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+        if (questioning) {
+            if (shouting)
+                return "Calm down, I know what I'm doing!";
+            return "Sure.";
+        }
+        if (shouting)
+            return "Whoa, chill out!";
+        return "Whatever.";
```
Comment (ErikSchierboom) [inline exercises/practice/bob/.approaches/if-statements/snippet.txt:8]:
```suggestion
if (questioning) {
    if (shouting)
        return "Calm down, I know what I'm doing!";
    return "Sure.";
}
if (shouting)
    return "Whoa, chill out!";
return "Whatever.";
```

**Review by ErikSchierboom — APPROVED** [review]
Just one indentation comment

**Comment by bobahop** [comment]
@ErikSchierboom: indentation suggestion accepted. Thanks for the catch! Ready for merge.

**Pull request merged**